src/main/java/org/reaktivity/specification/socks/internal/SocksFunctions.java

```diff
@@ -49,9 +55,22 @@ private SocksRouteExBuilder()
         }
 
         public SocksRouteExBuilder address(
-            String address)
+            String address) throws UnknownHostException
+        {
+            final InetAddress inet = InetAddress.getByName(address);
+            final byte[] ip = inet.getAddress();
+            final Consumer<Builder> addressBuilder = inet instanceof Inet4Address ?
+                b -> b.ipv4Address(s -> s.put(ip)):
+                b -> b.ipv6Address(s -> s.put(ip));
+
+            routeExRw.address(addressBuilder);
```

Comment:
Please follow naming convention for `routeExRW`, not `routeExRw`.

Reply:
Done